project.mml

```diff
@@ -157,8 +153,12 @@ Layer:
     geometry: polygon
     <<: *extents
     Datasource:
-      file: data/antarctica-icesheet-polygons-3857/icesheet_polygons.shp
-      type: shape
+      <<: *osm2pgsql
+      table: |-
+        (SELECT
+            way
+          FROM icesheet_polygons
+        ) AS icesheet_polygons
```

Comment:
Wouldn't `table: icesheet_polygons` work aswell? It's shorter

Reply:
Yes, but I'd rather have it consistent with other tables.